mpa/modules/models/heads/custom_hierarchical_linear_cls_head.py

```diff
@@ -95,12 +95,13 @@ def forward_train(self, x, gt_label, **kwargs):
         if self.compute_multilabel_loss:
             head_gt = gt_label[:, self.hierarchical_info['num_multiclass_heads']:]
             head_logits = cls_score[:, self.hierarchical_info['num_single_label_classes']:]
-            valid_mask = head_gt >= 0
-            head_gt = head_gt[valid_mask].view(*valid_mask.shape)
-            head_logits = head_logits[valid_mask].view(*valid_mask.shape)
+            valid_batch_mask = head_gt >= 0
+            head_gt = head_gt[valid_batch_mask, ]
```

Comment:
Does this mean head_gt[valid_batch_mask, :] ?

Reply:
Yes, it's the same